ansible/master.yml

```diff
@@ -8,6 +8,6 @@
         msg: 'Your Ansible version is lower than 2.8. Upgrade it.'
 
 - name: Configure beacon node builds
-  hosts: nimbus-master
+  hosts: nimbus-build-severs
```

Comment:
I suspect there's a hidden master host group list used by Terraform to generate this file and the group names come from there, so it's something to keep in mind.

Reply:
Yeah. This should probably be set to:
```yaml
  hosts:
    - master-large-01.aws-eu-central-1a.nimbus.pyrmont
    - master-small-01.aws-eu-central-1a.nimbus.pyrmont
```

Reply:
The groups are indeed generated from Terraform, as is the inventory file by [`ansible/terraform.py`](https://github.com/status-im/infra-nimbus/blob/master/ansible/terraform.py).